sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -1214,6 +1246,11 @@ object HiveExternalCatalog {
 
   val CREATED_SPARK_VERSION = SPARK_SQL_PREFIX + "create.version"
 
+  val TABLE_CONSTRAINT_PREFIX = SPARK_SQL_PREFIX + "constraint."
+  val TABLE_CONSTRAINT_PRIMARY_KEY = SPARK_SQL_PREFIX + TABLE_CONSTRAINT_PREFIX + "pk"
+  val TABLE_NUM_FK_CONSTRAINTS = SPARK_SQL_PREFIX + "numFkConstraints"
+  val TABLE_CONSTRAINT_FOREIGNKEY_PREFIX = SPARK_SQL_PREFIX + TABLE_CONSTRAINT_PREFIX + "fk."
```

Comment:
`SPARK_SQL_PREFIX` is duplicated in `TABLE_CONSTRAINT_PRIMARY_KEY` and `TABLE_CONSTRAINT_FOREIGNKEY_PREFIX`.

E.g., `TABLE_CONSTRAINT_PRIMARY_KEY` is `SPARK_SQL_PREFIX` + `SPARK_SQL_PREFIX` + "constraint.".